modules/prebidServerBidAdapter/index.js

```diff
@@ -371,6 +371,12 @@ function addBidderFirstPartyDataToRequest(request) {
       if (currBidderConfig.ortb2.user) {
         ortb2.user = currBidderConfig.ortb2.user;
       }
+      if (currBidderConfig.ortb2.bcat) {
+        ortb2.bcat = currBidderConfig.ortb2.bcat;
```

Comment:
Why not just merge the entire ortb2 object into the PBS request? A main point of that object was that we didn't have to weigh PBJS down with choosing which fields to send... just send the whole thing IMO.

Reply:
@bretg I agree with your approach, it makes things a lot more simple at PBJS side. Can you check my latest commit to see if that's what we wanted?

I have also removed the **if checks** for `bidderFirstPartyData`. Do wanna keep those checks for bidderFirstParyData?

Also, now, since there is no check in place, the publisher can pass any property downstream to PBS. I hope that's alright.